### ISSUE TITLE
YouHaveDied: ArgumentException: Mesh can not have more than 65000 vertices

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#661]
Auto bug report from 0.1.8.

Language: French

Feedback ID:  20201112-NVRV3

Need to check text components on the HighScoreCanvas – likely a shadow or bevel component creating too many vertices.

```
1:05:55, Frame 17959, EXCEPTION »  ArgumentException: Mesh can not have more than 65000 vertices
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

UnityEngine.UI.VertexHelper.FillMesh (UnityEngine.Mesh mesh)
UnityEngine.UI.Graphic.DoMeshGeneration ()
UnityEngine.UI.Graphic.UpdateGeometry ()
UnityEngine.UI.Text.UpdateGeometry ()
UnityEngine.UI.Text.FontTextureChanged ()
UnityEngine.UI.FontUpdateTracker.RebuildForFont (UnityEngine.Font f)
UnityEngine.Font.InvokeTextureRebuilt_Internal (UnityEngine.Font font)
UnityEngine.Canvas:SendWillRenderCanvases()

GameStates: RompingThroughDungeon, TransitioningFromGameOverToHighScores, ViewingHighScoreCanvas
GameSetupMode: Singleplayer
Players: 1
DungeonRoomTransitionerState: Null
Room: 6 / 6
RoomModifiers: DropGoldOnReceiveDamage (Full Pockets)
YouHaveDiedState: Visible
SystemPlayer InputDevice: Clavier
HumanPlayer 1 InputDevice: Clavier
```

**Comment by ChrisNZL** [comment]
ADDITIONALLY, High Score title's background streak is not getting resized when title changes.

**Comment by ChrisNZL** [comment]
Fixed in 0.1.8a.
